test/cmactest.c

```diff
@@ -177,19 +177,19 @@ static int test_cmac_run(void)
 
     ctx = CMAC_CTX_new();
 
-    /* Construct input data, fullfill data to match data length */
+    /* Construct input data, fulfill data to match data length */
```

Comment:
This should definitely be `repeat`

Reply:
Then `repeat` it is, It sounds good. Ill fix that.